### PR TITLE
Disable test based on 34178

### PR DESCRIPTION
**Pull request by jashook**

### CODE REVIEW

**Review by trylek — APPROVED** [review]
LGTM, thanks Jarret and have a great weekend! Hope your daughter's birthday was fine considering the circumstances.

File: src/coreclr/tests/issues.targets [hunk 1]
```diff
@@ -395,6 +395,9 @@
 
     <!-- Windows arm32 specific excludes -->
     <ItemGroup Condition="'$(XunitTestBinBase)' != '' and ('$(BuildArch)' == 'arm' or '$(AltJitArch)' == 'arm') and '$(TargetsWindows)' == 'true' and '$(RuntimeFlavor)' == 'coreclr'">
+        <ExcludeList Include="$(XunitTestBinBase)/JIT/Methodical/explicit/coverage/expl_float_1_*">
```
Comment (BruceForstall) [inline src/coreclr/tests/issues.targets]: Does that syntax work? You don't need `expl_float_1_*/*` or `expl_float_1_*/**`?

**Review by BruceForstall — COMMENTED** [review]
Looks like the exclusion didn't work since the tests still failed.

File: src/coreclr/tests/issues.targets (hunk 1 above)
Comment (trylek) [reply src/coreclr/tests/issues.targets]: Bruce is right, the path should always end with ** as the infra arbitrarily creates an extra subfolder level under the test folder.

Comment (trylek) [reply src/coreclr/tests/issues.targets]: That's what Nathan also fought when working on Mono-specific test exclusions.

**Comment by BruceForstall** [comment]
@jashook Are you going to be able to fix this?

**Comment by jashook** [comment]
Yes

**Pull request merged**